ports/espressif/common-hal/wifi/Radio.c

```diff
@@ -86,9 +86,7 @@ void common_hal_wifi_radio_set_enabled(wifi_radio_obj_t *self, bool enabled) {
     if (!self->started && enabled) {
         ESP_ERROR_CHECK(esp_wifi_start());
         self->started = true;
-        if (CIRCUITPY_WIFI_DEFAULT_TX_POWER != 20) {
-            common_hal_wifi_radio_set_tx_power(self, CIRCUITPY_WIFI_DEFAULT_TX_POWER);
-        }
+        common_hal_wifi_radio_set_tx_power(self, CIRCUITPY_WIFI_DEFAULT_TX_POWER);
```

Comment:
The check removes an unnecessary call at compile time (assuming optimizer is working). It was previously done with #if in your original PR.

Reply:
Well, maybe never mind, since it could be that in the future some other default was chosen.

Reply:
Oh I didn't know the optimizer caught these things. I'll keep it in mind next time.